batch/batch/front_end/front_end.py

```diff
@@ -612,6 +613,36 @@
     return web.Response()
 
 
+async def _delete_batch(app, batch_id, user):
+    db = app['db']
+
+    record = await db.select_and_fetchone(
+        '''
+SELECT closed FROM batches
+WHERE user = %s AND id = %s AND NOT deleted;
+''',
+        (user, batch_id))
+    if not record:
+        raise web.HTTPNotFound()
```

Comment:
In the spirit of idempotence I think we should always return 200 OK, even if a user tries to delete a batch that is not theirs.

Reply:
I see now that this is just a copy paste from below, we can address in a separate PR.